src/test/Makefile.am

```diff
@@ -181,6 +181,11 @@ UNITTEST_LDADD += \
 	-lsocket -lnsl
 endif
 
+unittest_ceph_crypto_SOURCES = test/ceph_crypto.cc
+unittest_ceph_crypto_LDADD = $(UNITTEST_LDADD) $(CEPH_GLOBAL)
+unittest_ceph_crypto_CXXFLAGS = $(UNITTEST_CXXFLAGS)
+check_TESTPROGRAMS += unittest_ceph_crypto
+
```

Comment:
@tchaikov This is the only `Makefile.am` change needed here? Also, do I have it in the right place within the file?